static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -35,7 +35,7 @@ const fillAdvertSlots = () => {
             config.get('isDotcomRendering', false) &&
             getBreakpoint() === 'mobile'
         // Get all ad slots
-        const adverts = qwery(dfpEnv.adSlotSelector)
+        const adverts = [...document.querySelectorAll(dfpEnv.adSlotSelector)]
```

Comment:
I think it's worth using `lib/$$` for consistency!

Reply:
Yep we're going to look at how to support $ and $$.

$$ does not support the use case of passing in a Bonzo element as the context which we'll need to support (or amend calling code) if we want to switch from $ to $$.

Looking at $$ it maintains an internal list of elements which might be a tad wasteful of memory when in most cases we just want to select elements from the dom.